polaris/evaluate/metrics/docking_metrics.py

```diff
@@ -0,0 +1,134 @@
+        mol_ref = dm.remove_stereochemistry(mol_ref)
+
+    if heavy_only:
+        mol_probe = dm.remove_hs(mol_probe)
+        mol_ref = dm.remove_hs(mol_ref)
+
+    # combine parameters
+    params = dict(
+        symmetrizeConjugatedTerminalGroups=symmetrizeConjugatedTerminalGroups, kabsch=kabsch, **params
+    )
+
+    # calculate RMSD
+    rmsd = _call_rdkit_rmsd(mol_probe, mol_ref, conf_id_probe, conf_id_ref, **params)
+    if not np.isnan(rmsd):
+        return rmsd
+
+    # try again but remove charges and hydrogens
+    mol_ref_uncharged = dm.to_neutral(dm.remove_hs(mol_ref))
+    mol_probe_uncharged = dm.to_neutral(dm.remove_hs(mol_probe))
+    rmsd = _call_rdkit_rmsd(mol_probe_uncharged, mol_ref_uncharged, conf_id_probe, conf_id_ref, **params)
+    if not np.isnan(rmsd):
+        return rmsd
+
+    # try again but neutralize atoms
+    mol_ref_neutralized = dm.to_neutral(mol_ref)
+    mol_probe_neutralized = dm.to_neutral(mol_probe)
+    rmsd = _call_rdkit_rmsd(mol_probe_neutralized, mol_ref_neutralized, conf_id_probe, conf_id_ref, **params)
+    if not np.isnan(rmsd):
+        return rmsd
+
+    # try again but on canonical tautomers
+    mol_ref_canonical = dm.canonical_tautomer(mol_ref)
+    mol_probe_canonical = dm.canonical_tautomer(mol_probe)
+    rmsd = _call_rdkit_rmsd(mol_probe_canonical, mol_ref_canonical, conf_id_probe, conf_id_ref, **params)
+    if not np.isnan(rmsd):
+        return rmsd
+
+    # try again but after neutralizing atoms
+    mol_ref_neutral_canonical = dm.canonical_tautomer(dm.to_neutral(mol_ref))
+    mol_probe_neutral_canonical = dm.canonical_tautomer(dm.to_neutral(mol_probe))
+    rmsd = _call_rdkit_rmsd(
+        mol_probe_neutral_canonical, mol_ref_neutral_canonical, conf_id_probe, conf_id_ref, **params
+    )
+    if not np.isnan(rmsd):
+        return rmsd
+
+    return np.nan
+
+
+def conformer_to_mol(mol, conf):
+    """conv"""
+    mol = dm.copy_mol(mol)
+    mol.RemoveAllConformers()
+    mol.AddConformer(conf, assignId=True)
+    return mol
+
+
+def rmsd_coverage(
+    y_pred: Union[str, List[dm.Mol]], y_true: Union[str, list[dm.Mol]], max_rsmd: float = 2, ignore_nan=False
+):
+    """
+    Calculate the coverage of molecules with an RMSD less than 2 Å compared to the reference molecule conformer.
+
+    Attributes:
+        mols_probe: List of predicted binding conformers.
+        mols_ref: List of ground truth binding confoermers.
+        max_rsmd: The threshold for determining acceptable rsmd.
+        ignore_nan: Ignore conformers that failed to compute RMSD.
+
+    """
+
+    if len(y_pred) != len(y_true):
+        assert ValueError(
+            f"The list of probing molecules and the list of reference molecules are different sizes. {len(y_pred)} != {len(y_true)} "
+        )
+
+    rmsds = np.array(
+        [robust_rmsd(mol_probe=mol_probe, mol_ref=mol_ref) for mol_probe, mol_ref in zip(y_pred, y_true)]
+    )
+
+    if ignore_nan:
+        rmsds = rmsds[~np.isnan(rmsds)]
```

Comment:
**Question:** Do we need this option? At a first glance, it seems to me you would never want to use this because it could give a distorted view of the results (e.g. all are NaN except for 1 - Coverage could be 100%, but is that meaningful?)

Reply:
I get your point. 
My initial thought was to provide an option that allows users to debug issues when RMSD discrepancies arise due to poor docking results or failures in RMSD computation.
But now I think the debugging should be handled at the user end.  

I will remove the option.